src/test/java/com/hashedin/redmask/integration/snowflake/RedMaskITTest.java

```diff
@@ -0,0 +1,410 @@
+package com.hashedin.redmask.integration.snowflake;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.hashedin.redmask.common.DataMasking;
+import com.hashedin.redmask.config.MaskConfiguration;
+import com.hashedin.redmask.exception.RedmaskConfigException;
+import com.hashedin.redmask.factory.DataBaseType;
+import com.hashedin.redmask.factory.DataMaskFactory;
+import org.apache.ibatis.jdbc.ScriptRunner;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+
+import static com.hashedin.redmask.integration.snowflake.RedMaskITUtils.createMaskingRuleVersionFive;
+import static com.hashedin.redmask.integration.snowflake.RedMaskITUtils.createMaskingRuleVersionFour;
+import static com.hashedin.redmask.integration.snowflake.RedMaskITUtils.createMaskingRuleVersionOne;
+import static com.hashedin.redmask.integration.snowflake.RedMaskITUtils.createMaskingRuleVersionSix;
+import static com.hashedin.redmask.integration.snowflake.RedMaskITUtils.createMaskingRuleVersionThree;
+import static com.hashedin.redmask.integration.snowflake.RedMaskITUtils.createMaskingRuleVersionTwo;
+
+/**
+ * To run the integration test remove @Ignore annotation and
+ * update Redshift credentials.
```

Comment:
Why Redshift?

Also add @Ignore here, otherwise tests will fail.

Reply:
changed and added